R/bin_time.R

```diff
@@ -223,7 +228,7 @@ bin_time <- function(occdf, min_ma = "min_ma", max_ma = "max_ma",
 
       # Remove mid_ma for fossil occurrences (if not already present as input)
       if (rmcol == TRUE) {
-        occdf <- occdf[, -which(colnames(occdf) == "mid_ma")]
+        occdf <- occdf[, -which(colnames(occdf) == "mid_ma"), drop = TRUE]
```

Comment:
I think this occdf will always have 2 or more columns (same as line 288), but it's probably fine to keep this?